src/nlp/consistency.jl

```diff
@@ -437,6 +437,64 @@ function consistent_functions(nlps; rtol=1.0e-8, exclude=[])
       end
     end
 
+    if !(jth_hess_coord in exclude)
+      k = m
```

Comment:
Was this going to be a `for`?

Reply:
It is true, they are not so useful.
Making another `for` and testing all the constraints hessian might become very expansive, what do you think?

Reply:
I'm happy with a single value being tested. @dpo, opinions?